Full version with Materials Cloud header and Plausible scripts

elsapassaro opened the pull request:
This introduces a new style version for the app, containing the Materials Cloud header bar and a script to collect statistics with Plausible. This can be used to provide a consistent graphical appearance without the need of iframing the apps.
The jinja header and the css are taken from the [materialscloud-org/frontend-theme](https://github.com/materialscloud-org/frontend-theme) repository.

The new style is set as default. This can be changed by setting the `X-App-Style` in the request header. With Apache, when forwarding, in a ReverseProxy section, one can add
`RequestHeader set X-App-Style standard`
or
`RequestHeader set X-App-Style lite`
to set the `standard` (with title & info) or `lite` (different style, no title) styles for the app.

giovannipizzi commented:
Thanks @elsapassaro ! Can you deploy one app on dokku (dev) with this so I can see how it looks like? (Or let me know how to check it live).

elsapassaro commented:
@giovannipizzi I've implemented it here: [https://layer-raman-ir.matcloud.xyz/](https://layer-raman-ir.matcloud.xyz/).
In order to make it work, I implemented the changes in this pull request: https://github.com/epfl-theos/tool-layer-raman-ir/pull/23

giovannipizzi reviewed (approved):
Thanks! Feel free to merge, create the images, and update everything (remember to document how to add one new tool to MC, including how to add the corresponding monitoring via plausible).
